server/controller/central_checkers_controller.go

```diff
@@ -0,0 +1,92 @@
+package controller
+
+import (
+	"context"
+)
+
+// Tile int
```

Comment:
Can you remove this comment?  And other comments that are redundant/unnecessary?

Reply:
I updated the comments!